web/themes/custom/cloudy/templates/content/construction_project/node--construction-project.html.twig

```diff
@@ -7,20 +7,32 @@
         {{ content.field_summary.0 }}
       </p>
       {{ content.field_featured_media }}
-      <div class="event__date">
-        <span class="d-flex align-items-center">
-        {% if content.field_display_date|render|striptags|trim is not empty %}
```

Comment:
While we wanted the time formatting, we still need the conditional display date. I'm adding this back in.